bin/varnishd/cache/cache_fetch.c

```diff
@@ -532,6 +532,12 @@ vbf_stp_startfetch(struct worker *wrk, struct busyobj *bo)
 		bo->uncacheable = 1;
 		wrk->handling = VCL_RET_DELIVER;
 	}
+	if (bo->do_pass && !bo->do_stream) {
+		/* Pass readahead should be disabled when not streaming. */
+		if (bo->fetch_objcore->boc->pass_prefetch > 0) {
+			bo->fetch_objcore->boc->pass_prefetch = 0;
```

Comment:
nit-picking: the `if()` serves no purpose

Reply:
Fixed.